WrightTools/artists/_colors.py

```diff
@@ -133,6 +134,158 @@ def make_colormap(seq, name="CustomMap", plot=False):
     return cmap
 
 
+def rotate_colormap(cmap, rotations, eps=0.0001):
+    """
+    From a LinearSegmentedColormap, rotate an integer number of times, and return
+    a new colormap.   Note:  these rotations are limited by the use of epsilon
+    and may cause errors if the number of rotations is very large.  The last index value
+    will be off by eps x rotations.   The method only works on tuples and arrays for segmentdata;
+    use the r parameter instead for make_cubehelix.   Not used for ListedColormaps.
+
+    Parameters
+    ----------
+    cmap : matplotlib.colors.LinearSegmentedColormap
+        Colormap to be cycled.
+    rotations : int
+        number of rotations
+    eps:  difference between end of first cycle and beginning of next cycle
+        which may need to be specified if the colormap has many index values
+
+    Returns
+    -------
+    matplotlib.colors.LinearSegmentedColormap
+
+    """
+    colors = cmap._segmentdata
+    newmap = cmap.copy()
+
+    if isinstance(colors["red"], tuple):
```

Comment:
Do I understand correctly that you are just changing the tuple into numpy array?  e.g. can you not just use
`reds = np.array([ri for ri in colors["red"])`?